pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/kits/KitTestUtil.java

```diff
@@ -513,5 +516,16 @@ public void setEsIndex(String esIndex) {
         new DDPInstanceDao().updateEsParticipantIndex(ddpInstanceId, esIndex);
     }
 
+
+    /**
+     * reads the test config and sets it as what dsm uses as config.
+     * This is needed for the tests that use values from config/secret to run
+     * */
+    public static void setDsmConfig() {
```

Comment:
It's a little weird to have a setter with no args.  If in all cases we always want to use the configManager, then the guts of this method should probably go into the KitTestUtil constructor so that they are always called.

Reply:
You'd be happy to know that I got rid of this completely